src/ld65/lineinfo.h

```diff
@@ -155,23 +155,27 @@ INLINE const char* GetSourceNameFromList (const Collection* LineInfos)
 /* Return the name of a source file from a list of line infos */
 {
     /* The relevant entry is in slot zero */
-    return GetSourceName (CollConstAt (LineInfos, 0));
+    return CollCount(LineInfos) ? GetSourceName (CollConstAt (LineInfos, 0)) :
+                                  "<unknown>";
```

Comment:
Looking at e.g. https://github.com/cc65/cc65/blob/master/src/ca65/lineinfo.c#L253 or https://github.com/cc65/cc65/blob/master/src/co65/o65.c#L386 it seems to me that "unknown" would better match the predominant style than "<unknown>".